src/Phansible/Roles/Php.php

```diff
@@ -57,6 +51,32 @@ public function transformValues(array $values, VagrantBundle $vagrantBundle)
         return $values;
     }
 
+    private function getPackagesMap($ppa)
+    {
+        $map_php5 = [
+            'mysql'   => 'php5-mysql',
+            'mariadb' => 'php5-mysql',
+            'pgsql'   => 'php5-pgsql',
+            'sqlite'  => 'php5-sqlite',
+            'mongodb' => 'php5-mongo',
+        ];
+        $map_php7 = [
+            'mysql'   => 'php7.0-mysql',
+            'mariadb' => 'php7.0-mysql',
+            'pgsql'   => 'php7.0-pgsql',
+            'sqlite'  => 'php7.0-sqlite3',
+            'mongodb' => 'php7.0-mongo',
+        ];
+
+        if (false !== strpos($ppa, 'php-7')) {
+            $map = $map_php7;
+        } else {
+            $map = $map_php5;
+        }
+
+        return $map;
```

Comment:
`return (false !== strpos($ppa, 'php-7')) ? $map_php7 : $map_php5;`

Would read better. But I'll accept as is too.